ev3dev/__init__.py

```diff
@@ -48,6 +48,9 @@ def get_current_platform():
     """
     board_info_dir = '/sys/class/board-info/'
```

Comment:
We should just use the existing "constant" on the `Device` class along with the path concatenation method, like we do for sensors/motors/etc. That would eliminate the need for special-casing it. With the code you have now, there'd have been no reason to update the submodule in the first place :laughing:

Reply:
I found that what got added in the submodule didn't work (there needs to be a board-info directory with specific structure, but board-info in the submodule is just a text file) and just wanted to get the CI tests passing for now.

Reply:
My bad, will fix this today.

Reply:
I don't have the EV3 around to test this myself; do I understand correctly there needs to be a `board-info/fake-board/uevent` file? And that should contain `fake`?

Reply:
yep, this is what it looks like on an ev3
```
robot@ev3[~]# tree /sys/class/board-info/
/sys/class/board-info/
└── board0 -> ../../devices/platform/board/board-info/board0

1 directory, 0 files
robot@ev3[~]# cat /sys/class/board-info/board0/uevent 
BOARD_INFO_HW_REV=7
BOARD_INFO_MODEL=LEGO MINDSTORMS EV3
BOARD_INFO_ROM_REV=6
BOARD_INFO_SERIAL_NUM=00165346201F
BOARD_INFO_TYPE=main
robot@ev3[~]# 
```

It needs to contain `BOARD_INFO_MODEL=FAKE-SYS`

Reply:
https://github.com/ddemidov/ev3dev-lang-fake-sys/commit/3cb2a092f8ae06b11f2faca2ecfecf92c92a9bfa